packages/api/src/routes/medical/schemas/patient.ts

```diff
@@ -27,11 +27,10 @@ export const driverLicenseIdentifierSchema = z.object({
 export const generalTypeIdentifierSchema = z.object({
   type: z.enum(generalTypes),
 });
-export const personalIdentifierSchema = basePersonalIdentifierSchema.merge(
-  driverLicenseIdentifierSchema
-);
-// TODO #369 reenable this when we manage to work with diff systems @ CW
-// .or(basePersonalIdentifierSchema.merge(generalTypeIdentifierSchema));
+export const personalIdentifierSchema = basePersonalIdentifierSchema
+  .merge(driverLicenseIdentifierSchema)
+  .or(basePersonalIdentifierSchema.merge(generalTypeIdentifierSchema));
+// TODO: Why is this type rebuilt in the api? It exists in both core and api-sdk
```

Comment:
Core came afterwards, so we prob wanted to have code on core that required this. I'd try to remove it from the API and keep it only on Core if we can.

Reply:
Done. Removed duplicate types.